modules/social_features/social_profile/modules/social_profile_manager_notes/social_profile_manager_notes.module

```diff
@@ -84,3 +87,34 @@ function social_profile_manager_notes_theme($existing, $type, $theme, $path) {
   ];
   return $items;
 }
+
+/**
+ * Implements hook_entity_view_display_alter().
+ */
+function social_profile_manager_notes_entity_view_display_alter(EntityViewDisplayInterface $display, array $context): void {
+  if ( $context['entity_type'] === 'profile'
```

Comment:
@YaroslavShandra88 Please fix the Coding Standards.

Reply:
> @YaroslavShandra88 Please fix the Coding Standards.

Done.